lib/structuraid_core/design_codes/schemas/rc/footings/one_way_shear_capacity_schema.rb

```diff
@@ -0,0 +1,23 @@
+module StructuraidCore
+  module DesignCodes
+    module Schemas
+      module RC
+        module Footings
+          class OneWayShearCapacitySchema
+            include DesignCodes::Utils::SchemaDefinition
+
+            required_params %i[
+              design_compression_strength
+              width
+              effective_height
+              capacity_reduction_factor
```

Comment:
I don't see you are using the reduction factor but in lib/structuraid_core/design_codes/nsr_10/rc/footings/one_way_shear_capacity.rb and lib/structuraid_core/design_codes/nsr_10/rc/footings/one_way_shear_capacity.rb. Does it make sense to have it required here?

Reply:
So as we discussed, I removed the reduction factor as a param